portfolio/src/main/java/com/google/sps/servlets/QuoteGenerator.java

```diff
@@ -9,10 +13,48 @@
 /** Handles requests sent to the /quote URL endpoint */
 @WebServlet("/quote")
 public class QuoteGenerator extends HttpServlet {
-
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>\"Life changes fast. Life changes in the instant.\" - Joan Didion</h1>");
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response)
+      throws IOException {
+    // Get a random quote
+    Quote randomQuote = this.getRandomQuote();
+
+    // Parse the quote and return the json output
+    final Gson gson = new Gson();
+    final String json = gson.toJson(randomQuote);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Build the quotes using the author name and quote text.
+   *
+   * @param authorName Name of the quote author
+   * @param quoteText  The quote text
+   * @return A Quote object with the author's name and the text
+   */
+  private Quote buildQuote(final String authorName, final String quoteText) {
+    final Quote quoteInfo = new Quote(authorName, quoteText);
+    return quoteInfo;
+  }
+
+  /**
+   * Return one random quote from an ArrayList of quotes
```

Comment:
nit: comment should be third person, so "Returns one quote...". Also best not to mention the data structure you are using in the comment. You might decide to change it later, and also the comment should focus on the functionality.

"Returns one random quote from a list of quotes" might be better